### PR TITLE
fix useraccount printcolumns

### PR DESCRIPTION
**Pull request by xcoulon**
retrieve 'Tier' from label, include 'Created_at' and remove 'NS_Limit'

see also:
- https://github.com/codeready-toolchain/member-operator/pull/418
- https://github.com/codeready-toolchain/toolchain-e2e/pull/678

Signed-off-by: Xavier Coulon <xcoulon@redhat.com>

### CODE REVIEW

**Comment by xcoulon** [comment]
from 
```
$ oc get useraccount 
NAME        NS LIMIT   TIER NAME   READY   REASON
johnsmith                          True    Provisioned
```
to 
```
$ oc get useraccount 
NAME        CREATED_AT             TIER           READY   REASON
johnsmith   2023-03-09T13:22:20Z   deactivate30   True    Provisioned
```
once the changes were applied on a temp cluster

**Review by ranakan19 — APPROVED** [review]
Looks good! Thanks!

File: api/v1alpha1/useraccount_types.go [hunk 1]
```diff
@@ -70,8 +70,8 @@ type UserAccountStatus struct {
 // +kubebuilder:subresource:status
 // +kubebuilder:resource:scope=Namespaced
 // +kubebuilder:printcolumn:name="User ID",type="string",JSONPath=`.spec.userID`,priority=1
-// +kubebuilder:printcolumn:name="NS Limit",type="string",JSONPath=`.spec.nsLimit`
```
Comment (mfrancisc) [inline api/v1alpha1/useraccount_types.go:73]: what was `nsLimit` supposed to contain?

Comment (xcoulon) [reply api/v1alpha1/useraccount_types.go:73]: I don't recall exactly, but the field was removed in https://github.com/codeready-toolchain/api/commit/dc40aec24fcf3b548bc50ded1cdb1bcb267eb284, almost 1 year ago, when we starting implementing the `space` and `spacebinding` resources.

Comment (mfrancisc) [reply api/v1alpha1/useraccount_types.go:73]: Ah never mind ! It doesn't exist I see 😄

**Review by mfrancisc — APPROVED** [review]
Nice! 👍

File: api/v1alpha1/useraccount_types.go (hunk 1 above)
Comment (mfrancisc) [reply api/v1alpha1/useraccount_types.go:73]: > I don't recall exactly, but the field was removed in https://github.com/codeready-toolchain/api/commit/dc40aec24fcf3b548bc50ded1cdb1bcb267eb284, almost 1 year ago, when we starting implementing the space and spacebinding resources.

Thanks for the extra details! 👍

Comment (alexeykazakov) [reply api/v1alpha1/useraccount_types.go:73]: `nsLimit` has never been used. Initially we intended to use it as the limit of namespaces users would be able to create on their own. We never implemented that capability.

**Review by alexeykazakov — APPROVED** [review]

File: api/v1alpha1/useraccount_types.go (hunk 1 above)
Comment (mfrancisc) [reply api/v1alpha1/useraccount_types.go:73]: Got it ! Thanks!

**Review by rajivnathan — APPROVED** [review]

**Pull request merged**